gen-man-pages: command added to output man pages for acbuild

dgonyeo opened the pull request:
With this commit "acbuild gen-man-pages" will create the directory
./man/ and output a tree of man pages into this directory.

Fixes https://github.com/appc/acbuild/issues/205.

dgonyeo commented:
Example output can be downloaded from here: https://csh.rit.edu/~dgonyeo/man-pages.tar.gz

jonboulle commented:
Would be nice if this could be done without being baked into the binary - how about hiding it behind a tag and having a build-docs script that builds it with that tag?

dgonyeo commented:
Just added the build tag `manpages` to `acbuild/man-page-generator.go`, and added the script `build-man-pages` that builds acbuild with that tag, calls `acbuild gen-man-pages`, and then deletes the compiled acbuild (so people don't accidentally start using the acbuild with man pages support).

jonboulle commented on build-man-pages:
```diff
@@ -0,0 +1,15 @@
+
+${DIR}/bin/acbuild gen-man-pages
+
+# Don't want to leave acbuild with the gen-man-pages support laying around
```
lying around

jonboulle commented:
LGTM, thanks!
